packages/apollo-federation-subgraph-compatibility/src/type-defs.ts

```diff
@@ -42,7 +42,7 @@ export const typeDefs = gql`
 
     directive @custom on OBJECT
 
-    type Product @custom @key(fields: "id") @key(fields: "sku package") @key(fields: "sku variation { id }") {
+    type Product @node @custom @key(fields: "id") @key(fields: "sku package") @key(fields: "sku variation { id }") {
```

Comment:
@darrellwarde can you verify the changes made here?

Reply:
No? 😆 It looks fine to me! It's a node in our database right?!

Reply:
It is!